plugins/cockroachdb_logs.yaml

```diff
@@ -0,0 +1,260 @@
+version: 0.0.1
+title: CockroachDB
+description: Log parser for CockroachDB
+parameters:
+  - name: enable_health_log
+    description: Enable to collect health logs
+    type: bool
+    default: true
+  - name: health_log_path
+    description: The absolute path to the CockroachDB health logs
+    type: "[]string"
+    default:
+      - "/var/log/cockroach-data/logs/cockroach-health.log"
+  - name: enable_dev_log
+    description: Enable to collect general Dev logs.
+    type: bool
+    default: true
+  - name: dev_log_path
+    description: The absolute path to the CockroachDB Dev Logs
+    type: "[]string"
+    default:
+      - "/var/log/cockroach-data/logs/cockroach.log"
+  - name: enable_error_log
+    description: Enable to collect stderr logs.
+    type: bool
+    default: true
+  - name: error_log_path
+    description: The absolute path to the CockroachDB stderr logs
+    type: "[]string"
+    default:
+      - "/var/log/cockroach-data/logs/cockroach-stderr.log"
+  - name: enable_sql_schema_log
+    description: Enable to collect sql schema logs.
+    type: bool
+    default: true
+  - name: sql_schema_log_path
+    description: The absolute path to the CockroachDB sql schema logs
+    type: "[]string"
+    default:
+      - "/var/log/cockroach-data/logs/cockroach-sql-schema.log"
+  - name: enable_telemetry_log
+    description: Enable to collect telemetry logs.
+    type: bool
+    default: true
+  - name: telemetry_log_path
+    description: The absolute path to the CockroachDB telemetry logs
+    type: "[]string"
+    default:
+      - "/var/log/cockroach-data/logs/cockroach-telemetry.log"
+  - name: enable_kv_distribution_log
+    description: Enable to collect kv distribution logs.
+    type: bool
+    default: true
+  - name: kv_distribution_log_path
+    description: The absolute path to the CockroachDB kv distribution logs
+    type: "[]string"
+    default:
+      - "/var/log/cockroach-data/logs/cockroach-kv-distribution.log"
+  - name: enable_pebble_log
+    description: Enable to collect cockroachdb pebble logs.
+    type: bool
+    default: true
+  - name: pebble_log_path
+    description: The absolute path to the CockroachDB pebble logs
+    type: "[]string"
+    default:
+      - "/var/log/cockroach-data/logs/cockroach-pebble.log"
+  - name: start_at
+    description: At startup, where to start reading logs from the file ('beginning' or 'end')
+    type: string
+    supported:
+      - beginning
+      - end
+    default: beginning
+  - name: timezone
+    description: Timezone to use when parsing the timestamp
+    type: timezone
+    default: UTC
```

Comment:
Can we add the [offset_storage_dir](https://github.com/observIQ/observiq-otel-collector/blob/release/v1.25.0/plugins/macos_logs.yaml#L30-L33) into here as well?

Reply:
Can we also add the `parse_to` and `retain_raw_logs` fields like we do [here](https://github.com/observIQ/observiq-otel-collector/blob/release/v1.25.0/plugins/mongodb_logs.yaml#L18-L28)